### PR TITLE
Fix OAuth Login flow (fixes #10)

### LINKED ISSUE #10: Error 400: redirect_uri_mismatch
I'm attempting the follow https://github.com/jamesread/prometheus-gmail-exporter#getting-client_secretsjson (as-is right now) but so far fail to be able to get this to work...

The problem I'm (now) facing is: 

_Access blocked: This app’s request is invalid. You can’t sign in because this app sent an invalid request. You can try again later, or contact the developer about this issue. [Learn more about this error](https://developers.google.com/identity/protocols/oauth2/web-server#authorization-errors-redirect-uri-mismatch). If you are a developer of this app, see error details: Request details: redirect_uri=http://example.com:50879/ Error 400: redirect_uri_mismatch_

The README is missing a note about having to set an _Authorized redirect URI_ when creating the _OAuth 2.0 Client ID_ - but that (alone) doesn't help, the fun part is that the port of `http://example.com:33657` it uses on the `redirect_uri` keeps changing... 🤣 

@jamesread just wanted to record this here for other future users; I'll try out the (quote #8) _[port of the python version](https://github.com/jamesread/prometheus-gmail-exporter/compare/python-online-auth) to do "online auth", rather than the "offline auth"_ branch next.

### PR DESCRIPTION
**Pull request by vorburger**
Fix to 9090 instead of random port, and default to localhost instead of example.com.

Fixes #10 (for me)

@jamesread

### CODE REVIEW

File: gmail-exporter.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def get_credentials():
         flow = InstalledAppFlow.from_client_secrets_file(args.clientSecretFile, SCOPES)
         flow.user_agent = 'prometheus-gmail-exporter'
 
-        credentials = flow.run_local_server(port=0, bind_addr = '0.0.0.0', host = args.oauthHost)
+        credentials = flow.run_local_server(port=9090, bind_addr = '0.0.0.0', host = args.oauthHost)
```
Comment (jamesread) [inline gmail-exporter.py:54]: Hey, why did you specify port 9090 here? By default the library picks a random port - I don't think that's a problem, but any other particular value in fixing it?

File: gmail-exporter.py [hunk 2]
```diff
@@ -257,7 +257,7 @@ def main():
     parser.add_argument('--clientSecretFile', default=get_homedir_filepath('client_secret.json'))
     parser.add_argument('--credentialsPath', default=get_homedir_filepath('login_cookie.dat'))
     parser.add_argument("--updateDelaySeconds", type=int, default=300)
-    parser.add_argument("--oauthHost", type=str, default="example.com")
+    parser.add_argument("--oauthHost", type=str, default="localhost")
```
Comment (jamesread) [inline gmail-exporter.py:260]: Yep this is probably a more sensible default.

**Review by jamesread — COMMENTED** [review]
Hey, thanks for the PR. I just have 1 question here.

I just pushed a change that gets in the way of this PR, but I'll try and fix that when I merge.

File: gmail-exporter.py (hunk 1 above)
Comment (vorburger) [reply gmail-exporter.py:54]: see #10... at least for me, it was impossible to get it to work with a random port! 

Only this made it possible (for me) to finally 😸  get further.

**Comment by jamesread** [comment]
Thanks for the description in #10 - accepting this (and sorry I pushed changes that clobbered your PR - I'll fix those).

**Pull request merged**